hack/markdownlint.sh

```diff
@@ -9,7 +9,9 @@ if [ "${IS_CONTAINER}" != "false" ]; then
   TOP_DIR="${1:-.}"
   find "${TOP_DIR}" \
        \( -path ./vendor -prune -o \
-       -path ./tools/operator-sdk -prune \) \
+       -path ./tools/operator-sdk -prune -o \
+       -path ./.github -prune \
```

Comment:
is -prune duplicated ? or is it expected after each path ?

Reply:
It's how I've seen it used elsewhere (including an earlier version of this file based on the diff).